src/platform/plugins/shared/dashboard/public/dashboard_api/layout_manager/layout_manager.ts

```diff
@@ -211,6 +212,11 @@ export function initializeLayoutManager(
             width: size?.width ?? DEFAULT_PANEL_WIDTH,
             height: size?.height ?? DEFAULT_PANEL_HEIGHT,
             currentPanels: layout$.value.panels,
+            /**
+             * We can assume that all panels being sent as a single package are related; so,
+             * place them close together by grouping them around the first embeddable.
+             */
+            beside: uuid === first.embeddableId ? undefined : first.embeddableId,
```

Comment:
Previously, if you had a controls-sized gap in your Dashboard, the controls would try to fill those gaps - which means they would be separated from the main histogram chart. By using the `beside` like this, we prevent that.